### PR TITLE
plir2:adding TX_CURR indicator

### PR DESCRIPTION
**Pull request by gitcliff**
Adding support for calculating TX_CURR indicator

### CODE REVIEW

**Comment by gitcliff** [comment]
@mozzy11 requesting for review 
Thanks

File: resources/tx_curr-Measure.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    "title": "Sample TX_CURR population selection criteria and evaluation",
+    "status": "draft",
+    "experimental": true,
+    "library": [
+        "Library/TX-CURR"
+    ],
+    "scoring": {
+        "coding": [
+            {
+                "code": "proportion"
```
Comment (mozzy11) [inline resources/tx_curr-Measure.json]: @gitcliff ,can you change this to the right measure scoring code, see https://www.hl7.org/fhir/valueset-measure-scoring.html

Comment (gitcliff) [reply resources/tx_curr-Measure.json]: @mozzy11 i have fixed it

Comment (HerbertYiga) [reply resources/tx_curr-Measure.json]: @mozzy11  is this now beautiful enough?

**Comment by mozzy11** [comment]
Thanks @gitcliff 
LGTM

File: resources/tx_curr-Measure.json (hunk 1 above)
Comment (mozzy11) [reply resources/tx_curr-Measure.json]: yes @HerbertYiga , it looks good. let me test and see

**Pull request merged**